src/app.js

```diff
@@ -120,9 +121,27 @@ export function app() {
 		request: function() {},
 		/**
 		 * Spawn an app...
-		 * @returns {Promise}
+		 * @param {string} target Target appId. - No wildcards supported
+		 * @param {string} topic Topic.
+		 * @param {*=} data Data.
+		 * @returns {Object} Chainable
 		 */
-		spawn: function() {},
+		spawn: function(target, topic, data = {}) {
```

Comment:
Silly question: Why are you not using method shorthand?

Reply:
We talked about making a decision and sticking with it. This is me deciding and going with it =)

I think it will be more clear in IDE's, once we have the type definitions up and people get autocomplete suggestions

Reply:
I completely misunderstood the question.


I didn't know that it was a standard feature.

I'll switch over right away =)